tools/accuracy_checker/configs/face-reidentification-retail-0095.yml

```diff
@@ -28,7 +28,7 @@ models:
       - name: lfw
         data_source: LFW/lfw
         annotation_conversion:
-          converter: face_reid_pairwise
+          converter: lfw
```

Comment:
How is this relevant to the rest of this PR?

Reply:
@eaidova is it correct?

Reply:
I'm sorry, I was going to update it in separated PR. These changes do not have a direct relation with adding new models.

Reply:
@IRDonch Should I revert this change by now?

Reply:
@eaidova I reverted this change in this PR